Riot/Categories/MXBugReportRestClient+Riot.swift

```diff
@@ -0,0 +1,117 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import MatrixSDK
+import GBDeviceInfo
+
+extension MXBugReportRestClient {
```

Comment:
@pixlwave I extracted the common common code for sending reports from the app in case it's useful for your needs.